tools/ci/commitGeneratedDocs.ps1

```diff
@@ -68,28 +68,23 @@ if (-not $output) {
 Write-Host "Destination folder: $DestFolder"
 
 # Clone the destination branch locally in a temporary folder.
-# Note that this creates a second copy of the repository inside itself.
 # This will be used to only commit changes to that gh-pages branch which
 # contains only generated documentation-related files, and not the code.
-# Note that we always clone into ".\_docs", which is the repository root,
+# Note that we always clone into $StagingFolder, which is the repository root,
 # even if the destination folder is a sub-folder.
 Write-Host "Clone the generated docs branch"
-$cloneCommand = "git -c http.extraheader=""AUTHORIZATION: $Authorization"" clone https://github.com/Microsoft/MixedReality-Sharing.git --branch gh-pages "".\_docs"""
+$cloneCommand = "git -c http.extraheader=""AUTHORIZATION: $Authorization"" clone https://github.com/Microsoft/MixedReality-Sharing.git --branch gh-pages ""$StagingFolder"" --no-checkout"
```

Comment:
@djee-ms the bug fixed by this change was caused by mis-copying the similar logic in MR-WebRTC. The code there cleared the new repo contents with
`Get-ChildItem "$DestFolder" -Recurse | Remove-Item -Force -Recurse`
which preserved the .git folder, while I used
`Remove-Item -Force -Recurse "$DestFolder`\*"`
which deleted it (thus breaking everything). I am not sure the original behavior is intended though 😄 For one thing, all the folders containing the docs for branches other than master are deleted when the docs for master are regenerated - this might be an oversight.